src/Neo/Network/P2P/Peer.cs

```diff
@@ -29,42 +29,28 @@ namespace Neo.Network.P2P
     /// </summary>
     public abstract class Peer : UntypedActor, IWithUnboundedStash
     {
-        public IStash Stash { get; set; }
+        public IStash Stash { get; set; } = null!;
```

Comment:
This is weird, it's because the interface?

```suggestion
        public IStash? Stash { get; set; }
```

Reply:
The system will automatically assign a value to it after the constructor, so we can assume that it is definitely not `null`.